### ISSUE TITLE
Subclassing is broken: ReferenceError: instance is not defined

### ISSUE THREAD

**Issue by ivan** [6to5/6to5#689]
I think 8d81a382f70f329e9a09dae0f8efab4dd5b73913 broke subclassing.

This code:

```js
class BaseClass {
	
}

class MyClass extends BaseClass {
	method() {
		return 3;
	}
}

var mc = new MyClass();
console.log(mc.method());
```

compiles to

```js
"use strict";

var _prototypeProperties = function (child, staticProps, instanceProps) { if (staticProps) Object.defineProperties(child, staticProps); if (instanceProps) Object.defineProperties(child.prototype, instanceProps); };

var _inherits = function (subClass, superClass) { if (typeof superClass !== "function" && superClass !== null) { throw new TypeError("Super expression must either be null or a function, not " + typeof superClass); } subClass.prototype = Object.create(superClass && superClass.prototype, { constructor: { value: subClass, enumerable: false, writable: true, configurable: true } }); if (superClass) subClass.__proto__ = superClass; };

var _classCallCheck = function (instance, Constructor) { if (!(instance instanceof Constructor)) { throw new TypeError("Cannot call a class as a function"); } };

var BaseClass = function BaseClass() {
        _classCallCheck(this, BaseClass);
};

var MyClass = (function (BaseClass) {
        function MyClass() {
                _classCallCheck(this, MyClass);

                if (Object.getPrototypeOf(MyClass) !== null) {
                        Object.getPrototypeOf(MyClass).apply(instance, arguments);
                }
        }

        _inherits(MyClass, BaseClass);

        _prototypeProperties(MyClass, null, {
                method: {
                        value: function method() {
                                return 3;
                        },
                        writable: true,
                        configurable: true
                }
        });

        return MyClass;
})(BaseClass);

var mc = new MyClass();
console.log(mc.method());
```

and throws

```
class MyClass extends BaseClass {
      ^
ReferenceError: instance is not defined
    at new MyClass (C:/Users/at/Documents/Trash/6to5fail.js:5:7)
    at Object.<anonymous> (C:/Users/at/Documents/Trash/6to5fail.js:11:10)
    at Module._compile (module.js:446:26)
    at normalLoader (D:\opt\NodeModules\6to5-core\lib\6to5\register.js:100:5)
    at Object.require.extensions.(anonymous function) [as .js] (D:\opt\NodeModules\6to5-core\lib\6to5\register.js:113:7)
    at Module.load (module.js:341:32)
    at Function.Module._load (module.js:296:12)
    at Function.Module.runMain (module.js:487:10)
    at Object.<anonymous> (D:\opt\NodeModules\6to5-core\bin\_6to5-node:93:12)
    at Module._compile (module.js:446:26)
```

**Comment by ivan** [comment]
Thanks!

**Comment by sebmck** [comment]
Fixed as of 3.3.9, thanks!
